client/prettier.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  printWidth: 80,
+  printWidth: 140,
```

Comment:
This little change in the prettier rules actually triggers a lot of changes from code formatting. From my point of view this reformatting is not needed for the actual update to Angular 11. It is hard for me to see, what changes are acually related to the Angular 11 update itself. Maybe the reformatting can be moved into a separate PR.

Reply:
I grouped the changes into several [commits](https://github.com/opendevstack/ods-provisioning-app/pull/632/commits), maybe it's easier to follow them that way?

Reply:
True true. Just was surprised by the amount of changed files 😜

Reply:
Totally understandable :-) But you were right, lint changes are not part of the update to 11 itself and would have been easier to follow in a separate PR.